docs/intro/tutorial05.txt

```diff
@@ -626,14 +626,14 @@ create a new test class for that view. It'll be very similar to what we have
 just created; in fact there will be a lot of repetition.
 
 We could also improve our application in other ways, adding tests along the
-way. For example, it's silly that ``Questions`` can be published on the site
-that have no ``Choices``. So, our views could check for this, and exclude such
-``Questions``. Our tests would create a ``Question`` without ``Choices`` and
+way. For example, it's silly that ``Question``\s can be published on the site
+that have no ``Choice``\s. So, our views could check for this, and exclude such
+``Question``\s. Our tests would create a ``Question`` without ``Choice``\s and
```

Comment:
I'm also replacing `silly` with `pointless` for the sake of being more inclusive (for neurodiverse crowd).
```suggestion
way. For example, it's pointless that a ``Question`` can be published on the site
that have no ``Choice`` instances. So, our views could check for this, and exclude such
``Question`` objects. Our tests would create a ``Question`` without any ``Choice`` and
```